packages/bpk-component-breadcrumb/readme.md

```diff
@@ -33,15 +40,17 @@ export default class App extends Component {
 
 ### BpkBreadcrumb
 
-| Property   | PropType | Required | Default Value |
-| ---------- | -------- | -------- | ------------- |
-| children   | node     | true     | -             |
+| Property           | PropType | Required | Default Value |
+| ------------------ | -------- | -------- | ------------- |
+| children           | node     | true     | -             |
+| accessibilityLabel | string   | true     | -             |
```

Comment:
I'm now wondering if this should just be called `label` or just stick with the underlying name `aria-label`?

`accessibilityLabel` is an RN prop name. In other web components we've opted for just `label` (see [BpkCloseButton.js](https://github.com/Skyscanner/backpack/blob/48739eb970403fe674707e760a8b220247560974/packages/bpk-component-close-button/src/BpkCloseButton.js))

Reply:
Flow is complaining when I use 'aria-current'. Lets go with just  'label'